index.html

```diff
@@ -245,10 +245,10 @@ <h3>RailsGirls Summer of Code</h3>
 					<h3>Outreachy</h3>
 					<p>3-month internships for people traditionally underrepresented in tech. Interns work remotely with mentors from Free and Open Source Software communities.</p>
 				</a>
-				<a class="col-md-4" href="https://www.redhat.com/en/about/women-in-open-source" target="_blank" rel="noopener">
-					<img alt="" src="img/projects/womeninopensource.png" />
-					<h3>Women in Open Source Award</h3>
-					<p>We believe open source is the future of technology. It's time to recognize the contributions women are making and inspire a new generation to join the open source movement.</p>
```

Comment:
Not really sure if you've removed this part intentionally.

Reply:
It was moved further down, so that all the 3 internship programs are in one row. :)